lib/time.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+function sleep(ms) {
```

Comment:
要不要考虑一下 https://www.npmjs.com/package/sleep

Reply:
这个库看过了，解的问题不一样。
这个 sleep 使用了 c++ binding，有一些注意事项：
> Note that because this is a C++ module, it will need to be built on the system you are going to use it on.

而且这个 sleep 是让整个 node sleep，和我们的需求不一样